src/System.Private.CoreLib/shared/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -48,9 +52,32 @@ public DefaultValueAttribute(Type type, string value)
                 {
                     _value = Convert.ChangeType(value, type, CultureInfo.InvariantCulture);
                 }
+
+                return;
+
+                // Looking for ad hoc created TypeDescriptor.ConvertFromInvariantString(Type, string)
+                bool TryConvertFromInvariantString(Type typeToConvert, string stringValue, out object conversionResult)
+                {
+                    conversionResult = null;
+
+                    // lazy init reflection objects
+                    if (s_convertFromInvariantString == null)
+                    {
+                        Type typeDescriptorType = Type.GetType("System.ComponentModel.TypeDescriptor, System, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089", throwOnError: false);
```

Comment:
This should load the type from System.ComponentModel.TypeConverter directly, not from System facade.

Reply:
> This should load the type from System.ComponentModel.TypeConverter directly, not from System facade.

Because we'll know if TypeConverter exists when we call conversion method and so check could pass but fail after?

Reply:
System.dll is .NET Framework compatibility shim. It should not be loaded or pulled unless really necessary

 System.ComponentModel.TypeConverter is the native .NET Core implementation assembly for this, so it is where we should be loading this from.

Reply:
Ahhh i misread...understood i used System...

Reply:
done